mtgjson5/output_generator.py

```diff
@@ -206,6 +207,13 @@ def build_all_printings_files(pretty_print: bool) -> None:
         pretty_print,
     )
 
+    # AllTcgSkus.json
```

Comment:
Was this profiled? I split up these calls intentionally to avoid breaking the 3.5GiB Mem usage. This might be better suited elsewhere.